test/acceptance/features/steps/steps.py

```diff
@@ -303,24 +307,34 @@ def quarkus_app_is_imported_as_knative_service(context, application_name):
 
 
 # STEP
-
-
 @then(u'"{app_name}" deployment must contain SBR name "{sbr_name}"')
 def then_envFrom_contains(context, app_name, sbr_name):
     time.sleep(60)
     openshift = Openshift()
+    secret = ""
```

Comment:
The whole steps need to be refined to something like:

```
{app_name}" deployment must contain reference to secret existing in service binding "{sbr_name}"
```

Reply:
Done here 4672da5

Reply:
Done here 4672da5